src/interfaces/IEthMultiVault.sol

```diff
@@ -566,6 +620,76 @@ interface IEthMultiVault {
     ///       See `getRedeemAssetsAndFees` for more details on the fees charged
     function redeemTriple(uint256 shares, address receiver, uint256 id) external returns (uint256);
 
+    /// @notice redeem shares from a bonding curve triple vault for assets
```

Comment:
3.10 - New *Curve Functions in EthMultiVault Missing From Interface

We have added the new curve related methods to the interface here.